sw/device/lib/testing/test_framework/ottf_isrs.c

```diff
@@ -0,0 +1,119 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/test_framework/ottf_isrs.h"
+
+#include "sw/device/lib/base/csr.h"
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/test_framework/FreeRTOSConfig.h"
+
+static uint32_t get_mtval(void) {
+  uint32_t mtval;
+  CSR_READ(CSR_REG_MTVAL, &mtval);
+  return mtval;
+}
+
+__attribute__((weak)) void ottf_exception_handler(void) {
```

Comment:
Nit: Can we lift `((weak))` into `OT_ATTR_WEAK` or something like that in macros.h?

Reply:
Fixed.